app/src/main/java/com/google/samples/apps/sunflower/viewmodels/GardenPlantingListViewModel.kt

```diff
@@ -17,16 +17,26 @@
 package com.google.samples.apps.sunflower.viewmodels
 
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import com.google.samples.apps.sunflower.data.GardenPlantingRepository
 import com.google.samples.apps.sunflower.data.PlantAndGardenPlantings
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.stateIn
 import javax.inject.Inject
 
 @HiltViewModel
 class GardenPlantingListViewModel @Inject internal constructor(
     gardenPlantingRepository: GardenPlantingRepository
 ) : ViewModel() {
-    val plantAndGardenPlantings: Flow<List<PlantAndGardenPlantings>> =
-        gardenPlantingRepository.getPlantedGardens()
+    val plantAndGardenPlantings: StateFlow<List<PlantAndGardenPlantings>> =
+        gardenPlantingRepository
+            .getPlantedGardens()
+            .stateIn(
+                viewModelScope,
+                SharingStarted.Lazily,
```

Comment:
You are collecting stateFlow with lifecycle,
https://github.com/android/sunflower/pull/976/files#diff-b218c68242277ca91d584e9db9dff7e2b1d5c32af496779fa559218ead47712eR68

but the SharingStarted.Lazily is never stop.
https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/-sharing-started/-companion/-lazily.html

Is this your intended behavior?

Reply:
@Jaehwa-Noh 
I corrected it, 
thank you again and if you have any other problems, please let me know.